src/Controls/tests/TestCases.HostApp/Issues/CarouselViewLoopNoFreeze.cs

```diff
@@ -13,7 +13,7 @@ public class CarouselViewLoopNoFreeze : ContentPage
 		readonly string _btnRemoveAllAutomationId = "btnRemoveAll";
 
 		readonly ViewModelIssue12574 _viewModel;
-		readonly CarouselView2 _carouselView;
+		readonly CarouselView1 _carouselView;
```

Comment:
If we do this, we lose the tests for CV2.

Since we got logic to have the system run this twice, can we drop the 1 or 2 and use the base CarouselView and let CI now run for both.

I think the original plan was to have a test for CV2 and not CV1 because CV1 was crashing, now you fixed it.